Symfony/CS/Fixer/Symfony/ScalarCastFixer.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ */
+final class ScalarCastFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        for ($index = 0, $count = $tokens->count(); $index  < $count; ++$index) {
+            if (!$tokens[$index]->isCast()) {
+                continue;
+            }
+
+            static $castMap = array(
+                'boolean' => 'bool',
+                'integer' => 'int',
+                'double' => 'float',
+                'real' => 'float',
+            );
+
+            $castFrom = substr($tokens[$index]->getContent(), 1, -1);
+            $castFrom = strtolower(trim($castFrom));
```

Comment:
Please do not hijack my comment ;P

In phpdoc the case may matter since one may have class `Boolean`. But casting is case-insensitive